compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract/Templates.hs

```diff
@@ -150,51 +147,6 @@ isChoiceTy ty
 
   | otherwise = Nothing
 
--- | If the given instance declaration is declaring an interface choice instance, return interface
--- name and choice name.
-isIfaceChoice :: ClsInstDecl GhcPs -> Maybe (Typename, Typename)
-isIfaceChoice (XClsInstDecl _) = Nothing
-isIfaceChoice decl@ClsInstDecl{}
-  | Just (ifaceName, ty) <- hasImplementsConstraint decl
```

Comment:
Now that Interface choices are monomorphic (https://github.com/digital-asset/daml/pull/14853), they are no longer declared using `Implements` constraints. How do we find them, then? turns out `isChoice`, previously only for template choices, works out of the box with the new representation of interface choices since it also relies on `HasExercise` instances.